test/integration/create-next-app/prompts.test.ts

```diff
@@ -250,4 +255,45 @@ describe('create-next-app prompts', () => {
       })
     })
   })
+
+  it('should not create app if --dry-run', async () => {
+    const projectName = 'dry-run'
+    await useTempDir(async (cwd) => {
+      const childProcess = createNextApp(
+        [
+          projectName,
+          '--ts',
+          '--app',
+          '--eslint',
+          '--no-turbo',
+          '--no-src-dir',
+          '--no-tailwind',
+          '--no-import-alias',
+          '--dry-run',
+        ],
+        {
+          cwd,
+        },
+        nextTgzFilename
+      )
+      await new Promise<void>(async (resolve) => {
+        childProcess.on('exit', async (exitCode) => {
+          expect(exitCode).toBe(0)
+          projectFilesShouldNotExist({
+            cwd,
+            projectName,
+            files: ['package.json'],
+          })
+          resolve()
+        })
+        let output = ''
+        childProcess.stdout.on('data', (data) => {
+          output += data
+          process.stdout.write(data)
+        })
+        await check(() => output, /Running a dry run, skipping installation/)
+        await check(() => output, /Dry Run Result/)
```

Comment:
Can you include here what the output will look like? This test isn't actually testing that we output anything useful.

Reply:
P.S. The changes look a lot since the upper stack was merged.
x-ref: https://github.com/vercel/next.js/pull/68246

Reply:
Couldn't use inline snapshot since the `appPath` may vary on device.